src/Uno.Foundation/Metadata/ApiInformation.cs

```diff
@@ -52,6 +52,9 @@ public static bool IsTypePresent(string typeName)
 		}
 	}
 
+	public static bool IsMethodPresent(Type type, string methodName)
```

Comment:
Good update, but let's keep those internal, as they're not part of the public API.